demo/Google.Storage.V1.Demo/BucketHelpers.cs

```diff
@@ -0,0 +1,122 @@
+// Copyright 2015 Google Inc. All Rights Reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Google.Storage.V1.Demo
+{
+    public static class BucketHelper
+    {
```

Comment:
I think they'd be useful in the lib itself, yes.

In demo/Google.Storage.V1.Demo/BucketHelpers.cs<https://github.com/GoogleCloudPlatform/gcloud-dotnet/pull/118#discussion_r60028604>:

> +//     http://www.apache.org/licenses/LICENSE-2.0
> +//
> +// Unless required by applicable law or agreed to in writing, software
> +// distributed under the License is distributed on an "AS IS" BASIS,
> +// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
> +// See the License for the specific language governing permissions and
> +// limitations under the License.
> +
> +using System;
> +using System.Collections.Generic;
> +using System.Linq;
> +
> +namespace Google.Storage.V1.Demo
> +{
> +    public static class BucketHelper
> +    {


Would I be right in thinking you probably want these within the library itself at some point?

-
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub<https://github.com/GoogleCloudPlatform/gcloud-dotnet/pull/118/files/f2279a6807fb057107949cbabca036d0ced8a75a#r60028604>